lib/defaults/defaults.go

```diff
@@ -597,6 +597,11 @@ func KubeProxyListenAddr() *utils.NetAddr {
 	return makeAddr(BindIP, KubeListenPort)
 }
 
+// WindowsDesktopProxyListenAddr returns the default listening address for the Desktop Proxy service
+func WindowsDesktopProxyListenAddr() *utils.NetAddr {
```

Comment:
```suggestion
func WindowsDesktopServiceListenAddr() *utils.NetAddr {
```

Can we call it "Windows Desktop Service" instead of proxy?